setup.py

```diff
@@ -16,7 +16,7 @@
 with open(here / "README.md", encoding="utf-8") as f:
     readme = f.read()
 
-dev_requires = ["black>=22.3.0", "pre-commit", "devtools", "jupyter", "flake8", "pyarrow"]
+dev_requires = ["black>=22.3.0", "pre-commit", "devtools", "flake8", "pyarrow"]
```

Comment:
I got an out-of-space error in CI for some reason and removed jupyter. Not sure if that was useful, but we can let the user opt-in for that.